action.yml

```diff
@@ -0,0 +1,37 @@
+name: Add pull request to project
+description: GitHub Action to add new pull requests to the Community Contributions GitHub project
+
+runs:
+  using: composite
+  steps:
+
+    - name: Check if should be added
+      id: check-if-should-add
+      env:
+        PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+      run: |
+        # We don't want CMS Squad member PRs to clutter the project board
+        echo "Author is $PR_AUTHOR"
+        for squad_member in 'GuySartorelli' 'emteknetnz' 'maxime-rainville'; do
+          if [[ $PR_AUTHOR == $squad_member ]]; then
+            echo "Author is in CMS Squad. Skipping."
+            echo "should_add_to_project=false" >> $GITHUB_OUTPUT
+            exit 0
```

Comment:
Doesn't `exit 0` still go to the next step? Or am I getting confused

We've got an output variable which we're using as a conditional in the generate-token step - I guess we'd also need that condition in the "Add to project" step?

Reply:
> Doesn't `exit 0` still go to the next step? Or am I getting confused

`exit 0` moves to the next step, which is what the condition is for. We do not want to exit 1 because we don't want the action to be marked red. It's not red, just skipped.

> We've got an output variable which we're using as a conditional in the generate-token step - I guess we'd also need that condition in the "Add to project" step?

I'll add it to that step.

Reply:
Done